lib/git-fastclone.rb

```diff
@@ -47,13 +47,23 @@ def reference_repo_dir(url, reference_dir, using_local_repo)
     end
     module_function :reference_repo_dir
 
+    def reference_filename(filename)
+      portation_character = if RbConfig::CONFIG['host_os'] =~ /mswin|msys|mingw|cygwin|bccwin|wince|emc/
+                              '_'
+                            else
+                              ':'
+                            end
+      "#{portation_character}#{filename}"
+    end
+    module_function :reference_filename
+
     def reference_repo_submodule_file(url, reference_dir, using_local_repo)
-      "#{reference_repo_dir(url, reference_dir, using_local_repo)}:submodules.txt"
+      "#{reference_repo_dir(url, reference_dir, using_local_repo)}#{reference_filename('submodules.txt')}"
     end
     module_function :reference_repo_submodule_file
 
     def reference_repo_lock_file(url, reference_dir, using_local_repo)
-      lock_file_name = "#{reference_repo_dir(url, reference_dir, using_local_repo)}:lock"
+      lock_file_name = "#{reference_repo_dir(url, reference_dir, using_local_repo)}#{reference_filename('lock')}" # rubocop:disable Layout/LineLength
```

Comment:
The modern rubocop default is 120 for this. I think I'd rather we delete [this](https://github.com/square/git-fastclone/blob/master/.rubocop.yml#L19) config instead and let it pick up the default rather than adding a `rubocop:disable` for something relatively trivial like this.

Reply:
I removed the LineLength rule from robucop config, and removed the useless disable rule on the code, double check it should be resolved by now